assets/src/stories-editor/blocks/amp-story-text/edit.js

```diff
@@ -42,21 +42,24 @@ class TextBlockEdit extends Component {
 			width,
 			content,
 			ampFitText,
+			ampFontFamily,
 		} = attributes;
 
-		const checkFontSize = (
+		const checkFontSize = ampFitText && (
```

Comment:
^ This check is actually already within `maybeUpdateFontSize` but I guess it doesn't harm to have it here + makes it more clear/readable, too.